cardinal/system/system_tokeniser.go

```diff
@@ -100,58 +107,98 @@ func (ts *TokeniserSystem) setupVrbAct() {
 	ts.responseLookup[enums.ActionTypeBreak] = []enums.ActionType{enums.ActionTypeBreak}
 }
 
+// setupGrammar initializes the grammar response tookup table with predifined grammar
+func (ts *TokeniserSystem) setupGrammar() {
+	ts.grammarLookup["THE"] = enums.GrammarTypeDefinitionArticle
+	ts.grammarLookup["TO"] = enums.GrammarTypePreposition
+	ts.grammarLookup["AT"] = enums.GrammarTypePrepo
+	ts.grammarLookup["Around"] = enums.GrammarTypeAdverb
+}
+
 // FishTokens processes the tokenized command and returns VerbData
 func (ts *TokeniserSystem) FishTokens(tokens []string) component.VerbData {
 	var data component.VerbData // Initialize VerbData to store the result
-	var err uint8               // Error code variable
+	var err uint8 = 0           // Error code variable
 	lenTokens := len(tokens) - 1
 
 	// Look up the verb, object, and directional object from the tokens
-	vrb, vrbExists := ts.cmdLookup[strings.ToUpper(tokens[0])]
-	obj, objExists := ts.objLookup[strings.ToUpper(tokens[lenTokens])]
-	dobj, dobjExists := ts.dirObjLookup[strings.ToUpper(tokens[lenTokens])]
-
-	if !vrbExists {
-		// Handle the case where the verb lookup fails
-		fmt.Printf("Error: Verb '%s' not recognized\n", tokens[0])
-		data.ErrCode = 1 // Set an appropriate error code
-		return data
-	}
+	var vrb enums.ActionType = ts.cmdLookup[(tokens[0])]
+	var obj enums.ObjectType = ts.objLookup[(tokens[lenTokens])]
+	var dobj enums.DirObjectType = ts.dirObjLookup[(tokens[lenTokens])]
```

Comment:
this should be an INDIRECTOBJECT, remember there is NO OBJECT it makes no sense to the tokeniser: the tokeniser looks for ACTION/VERB, DOBJ and IOBJ. kick the ball. v:kick, do:ball, io:none. "kick ball at window": v:kick, do:ball, io:window

Reply:
done